<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialUpWidget</class>
 <widget class="QWidget" name="DialUpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="actionLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>142</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Action:</string>
       </property>
       <property name="buddy">
        <cstring>actionComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="actionComboBox">
       <property name="currentText">
        <string>Create</string>
       </property>
       <item>
        <property name="text">
         <string>Create</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Replace</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Update</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Delete</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="locationFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="userRadioButton">
        <property name="text">
         <string>User connection</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">driveButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="systemRadioButton">
        <property name="text">
         <string>All users connection</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">driveButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="connectionLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Connection Name:</string>
        </property>
        <property name="buddy">
         <cstring>connectionLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="connectionLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="connectionToolButton">
        <property name="font">
         <font>
          <family>Sans Serif</family>
         </font>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="phoneLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Phone number:</string>
        </property>
        <property name="buddy">
         <cstring>phoneLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="phoneLineEdit">
        <property name="font">
         <font>
          <family>Sans Serif</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>actionComboBox</tabstop>
  <tabstop>userRadioButton</tabstop>
  <tabstop>systemRadioButton</tabstop>
  <tabstop>connectionLineEdit</tabstop>
  <tabstop>connectionToolButton</tabstop>
  <tabstop>phoneLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="driveButtonGroup"/>
 </buttongroups>
</ui>
